### DISCUSSION TITLE
What is the commit frequency?

### DISCUSSION THREAD

**Discussion by littlegolden** [rob006-software/flarum-translations#566] [General]
After weblate submits the latest translation to this repository, how long does it take to synchronize to the language pack repository?

Based on past records, it seems to be within 10 minutes.

**Comment by rob006** [comment]
Cron job for exporting translations to language pack runs every 10 minutes. So it mostly depends on timing, but it should be no more than 10 minutes.

**Comment by rob006** [comment]
I explained this in [FAQ](https://github.com/rob006-software/flarum-translations/wiki#how-long-does-it-take-for-the-text-translated-in-weblate-to-appear-in-the-language-pack).
